index.html

```diff
@@ -17,10 +18,126 @@
         <section class="main">
             <h1>Lekan Jimoh</h1>
             <p>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
-            <br>If you like what you see and have a project you need coded, don’t hesitate and contact me.
+            <br>If you like what you see and have a project you need coded, don't hesitate and contact me.
             </p>
-            <button type="button"> Start collaboration</button>
+            <button class="get-cv" type="button"> Start collaboration</button>
         </section>
         <img class="background" src="img/logo1.svg" alt="layout">
+        <section class="container">
+            <section class="layout-2">
+                <h1>Projects</h1>
+                <img src="img/laptop.svg" alt="project1">
+                <div class="wrapper">
+                    <h2>Project Name goes there</h2>
+                    <div class="tech-button">
+                        <button class="item" type="button">HTML/CSS</button>
+                        <button class="item" type="button">Ruby on Rails</button>
+                        <button class="item" type="button">JavaScript</button>
```

Comment:
- [ ] [OPTIONAL] These are supposed to be just `ul` `li` instead of buttons since you just want to list the technologies you used, there won't be any functionality for them later on.